src/lib/wpack.js

```diff
@@ -55,14 +55,16 @@ const createConfigs = (fns, config, servicePath, defaultOutput, folder) =>
 /**
  * Runs webpack with an array of configurations
  * @param {array} configs Array of webpack configurations
+ * @param {any} webpackConfig base webpack config
  * @returns {Promise} Webpack stats
  */
-const run = configs =>
+const run = (configs, webpackConfig) =>
```

Comment:
Can't we use `configs[0]` here instead of passing the base config?

Reply:
It seemed to me that using the base one would be a cleaner solution as configs are being transformed and you can't be sure if the part you're interested in wasn't modified. Atm only entry and output is being alternated but this may change. It's not a big deal though so if you prefer `configs[0]` let's do it your way :)